pkg/models/token.go

```diff
@@ -0,0 +1,7 @@
+package models
+
+type TokenResponse struct {
+	ID    string `json:"id"`
+	Token string `json:"token"`
+	Name  string `json:"name"`
+}
```

Comment:
Perhaps it could be a `CustomTokenResponse`?

Reply:
changed name